templates/tutorial/client/src/views/control/MapLegend.js

```diff
@@ -37,6 +40,13 @@ export const MapLegend = component(
             value={layerVisibility["ctl"]}
             onChange={value => dispatch(mapEvent.CHANGE_LAYER_VISIBILITY, ["ctl", value])}
           />
+          {isScenarioSelected &&
+           <Toggle
+             label={"Dark mode"}
+             value={isDarkMode}
+             onChange={value => dispatch(appEvent.SET_DARK_MODE, value)}
```

Comment:
Is there any particular reason to create a specific event for this rather than just dispatching:
```
scenarioEvent.CHANGE_CURRENT_VALUE, [['isDarkMode'], value]
```

Reply:
https://github.com/mit-cave-open/cave-app/pull/2/files#diff-a39c9f7aa728d5fa3b973bc6ba49228aR1607